device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsDeviceAuthenticationCBS.java

```diff
@@ -26,8 +26,8 @@
     public static final String SENDER_LINK_ENDPOINT_PATH = "$cbs";
     public static final String RECEIVER_LINK_ENDPOINT_PATH = "$cbs";
 
-    private static final String SENDER_LINK_TAG_PREFIX = "cbs-sender-";
-    private static final String RECEIVER_LINK_TAG_PREFIX = "cbs-receiver-";
+    public static final String SENDER_LINK_TAG_PREFIX = "cbs-sender-";
```

Comment:
Is changing of scope required?

Reply:
I'm working on a refactor where this can stay private